core/src/main/java/io/seata/core/exception/TransactionExceptionCode.java

```diff
@@ -114,6 +114,11 @@
      * FailedWriteSession
      */
     FailedWriteSession,
+
+    /**
+     * Failed to store exception code
+     */
+    FailedStore
     ;
 
     private static final Map<Integer, TransactionExceptionCode> MAP = new HashMap<>(values().length);
```

Comment:
values().length*2 to avoid resize of HashMap.

Reply:
This time there is no change here, is it necessary to modify it in this submission?

Reply:
> This time there is no change here, is it necessary to modify it in this submission?

Please amend it by the way.

Reply:
ok